### PR TITLE
[SCB-2582]bump apache version from 21 to 25

### PR DESCRIPTION
**Pull request by Matrix0xCC**
1. bump `apache` version from `21` to `25`
3. fix broken test cases caused by duplicate class `TestNullFieldAndDefaultValueParam`

### CODE REVIEW

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java [hunk 1]
```diff
@@ -43,7 +43,12 @@ public MicroserviceDeployDefinition getMicroserviceDeployDefinition() {
   @Override
   protected String[] createCmds() {
     // must set jar at the end of the cmds
-    return new String[] {"java", "-jar"};
+    return new String[] {"java", "-jar",
```
Comment (Shoothzj) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java:46]: I suggest don't commit this change if not needed.

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java [hunk 2]
```diff
@@ -43,7 +43,12 @@ public MicroserviceDeployDefinition getMicroserviceDeployDefinition() {
...
+        //add a property definition to enable quick search in taskmgr
+        "-DCombDeploy=true",
```
Comment (Shoothzj) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java:48]: can we use jps to find the process?

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java [hunk 3]
```diff
@@ -43,7 +43,12 @@ public MicroserviceDeployDefinition getMicroserviceDeployDefinition() {
...
+        //remote debug, default disabled
+        //"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=0.0.0.0:" + RandomUtils.nextInt(60000, 61000)
```
Comment (Shoothzj) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java:50]: commented out code. we can make it a config in Deploy like heap memory config.

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java (hunk 2 above)
Comment (Matrix0xCC) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java]: It's just a marker to search in windows task manager, I'll remove it later.

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java (hunk 3 above)
Comment (Matrix0xCC) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java]: removed

**Review by Shoothzj — APPROVED** [review]

**Pull request merged**